### DISCUSSION TITLE
Git clone missing in Open Telemetry data

### DISCUSSION THREAD

**Discussion by astellingwerf** [renovatebot/renovate#23902] [Suggest an Idea]
### How are you running Renovate?

Self-hosted Renovate

### If you're self-hosting Renovate, tell us what version of Renovate you run.

36.42.4

### If you're self-hosting Renovate, select which platform you are using.

GitLab self-hosted

### Was this something which used to work for you, and then stopped?

I am trying to get this working for the first time

### Describe the problem

It would help to understand where time is spend during a Renovate scan if the `git clone` command is also represented by a span. There was a project for which the block timeout was reached, after two minutes, I guess, because the Jaeger UI view of it looks like:
![image](https://github.com/renovatebot/renovate/assets/1637358/95871743-a45b-42bb-9b59-8fe3106da2f7)

### Relevant debug logs

<details><summary>Logs</summary>

```
Copy/paste the relevant log(s) here, between the starting and ending backticks
```

</details>

### Have you created a minimal reproduction repository?

Placeholder value, please select the correct response from the dropdown

**Comment by rarkins** [comment]
Feature request, not a bug?

**Reply by astellingwerf** [reply]
Yeah, sorry @rarkins, I don't know where my head was. Let's say it's been a while since I did this...

**Comment by jamietanna** [comment]
This is now available
